src/components/Movie/Movie.css

```diff
@@ -0,0 +1,23 @@
+
+  .movie-poster {
+    border-radius: 1em;
+    box-shadow: 7px 2px 20px 0px;
+    /* padding: 10px 10px 10px 10px; */
+    width: 24vw;
+    height: auto;
+    overflow: hidden;
+    object-fit: cover;
+  }
+
+.poster-container {
+    display: flex;
+    flex-direction: column;
```

Comment:
Perhaps we can fix our button issue by having more specific css rules here. Let's experiment with align-items: flex-end, justify-content: flex-end, and go from there. We may also need to edit our grid display to keep our cards from moving with every flex rule.